scripts/gee

```diff
@@ -82,6 +82,55 @@ review.
 
      gee rupdate
 
+## Environment variables
+
+`gee`'s behavior can be moderated by setting the following variables in your shell environment:
+
+* `GHUSER`: Your github username.
+
+* `GEE_DIR`: By default, gee will place all work directories in `~/gee`.  This variable can
+  be set to a different path to override this behavior.  `gee` will still create a repository
+  directory beneath `$GEE_DIR` (ie, `~/gee/internal`), so if you want to change the repository
+  directory as well, use the GEE_REPO and GEE_REPO_DIR environment variables.
+
+* `GEE_REPO_DIR`: By default, gee will place a repository's workdirs in a directory specified
+  by `${GEE_DIR}/${GEE_REPO}` where `GEE_DIR` defaults to `~/gee` and `GEE_REPO` defaults to `internal`.
+  The `GEE_REPO_DIR` can be set to specify a different directory.  However, if `GEE_REPO_DIR` is set,
+  you must also set the `GEE_REPO` environment variable.
+
+* `GEE_REPO`: By default, gee will attempt to guess which repository to use based on the current
+  working directory (ie. the `enkit` repo is inferred from `~/gee/enkit/master`).  To override this
+  behavior (or, if you are using `GEE_REPO_DIR` to use a non-standard directory structure), set
+  the `GEE_REPO` variable to the repository name (ie. `internal`, `enkit`, etc.).
+
+* `UPSTREAM`: The name of the github user hosting the specified repository.  By default, `enfabrica`.
```

Comment:
Is there a reason for not prefixing this with `GEE_`?

Reply:
These aren't new environment variables, and at this point I don't want to change the existing behavior.